### PR TITLE
feat: add rules ks-admission-general-block-nodeport-services&ks-admission-general-container-resource-limits

### PR DESCRIPTION
**Pull request by ComradeProgrammer**
feat: add rules ks-admission-general-block-nodeport-services&ks-admission-general-container-resource-limits

### CODE REVIEW

**Comment by ComradeProgrammer** [comment]
part of the implementation of #4

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    //"go.testFlags": ["-v"]
```
Comment (sagilio) [inline .vscode/settings.json:2]: `.vscode` directory should be added in `.gitignore`.

File: example/casbinconfig/allowed_repo.conf [hunk 2]
```diff
@@ -8,4 +8,4 @@ p = namespace,image,eft
 e=!some(where (p.eft == deny))
 
 [matchers]
-m= r.namespace==p.namespace &&r.image==p.image
\ No newline at end of file
+m= r.namespace == p.namespace && r.image == p.image
```
Comment (sagilio) [inline example/casbinconfig/allowed_repo.conf:11]: `m = r.namespace == p.namespace && r.image == p.image`

File: example/casbinconfig/container_resource_limit.conf [hunk 3]
```diff
@@ -0,0 +1,11 @@
+[request_definition]
+r = namespace, cpu, memory
+
+[policy_definition]
+p = namespace, cpu, memory, eft
+
+[policy_effect]
+e=some(where (p.eft == allow))
```
Comment (sagilio) [inline example/casbinconfig/container_resource_limit.conf:8]: e = some(where (p.eft == allow))

File: example/casbinconfig/container_resource_limit.conf [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+[matchers]
+m= r.namespace == p.namespace && r.cpu < parseFloat(p.cpu) && r.memory < parseFloat(p.memory)
```
Comment (sagilio) [inline example/casbinconfig/container_resource_limit.conf:11]: `m = r.namespace == p.namespace && r.cpu < parseFloat(p.cpu) && r.memory < parseFloat(p.memory)`

File: .vscode/settings.json (hunk 1 above)
Comment (sagilio) [inline .vscode/settings.json:2]: `.vscode` should be added in `.gitignore`

File: example/casbinconfig/allowed_repo.conf (hunk 2 above)
Comment (sagilio) [inline example/casbinconfig/allowed_repo.conf:11]: `m = r.namespace == p.namespace && r.image == p.image `

File: example/casbinconfig/container_resource_limit.conf (hunk 3 above)
Comment (sagilio) [inline example/casbinconfig/container_resource_limit.conf:8]: `e = some(where (p.eft == allow))`

File: example/casbinconfig/container_resource_limit.conf (hunk 4 above)
Comment (sagilio) [inline example/casbinconfig/container_resource_limit.conf:11]: `m = r.namespace == p.namespace && r.cpu < parseFloat(p.cpu) && r.memory < parseFloat(p.memory)`

File: example/casbinconfig/container_resource_limit.csv [hunk 5]
```diff
@@ -0,0 +1 @@
+p,default,104857600,104857600,allow
```
Comment (sagilio) [inline example/casbinconfig/container_resource_limit.csv:1]: `p, default, 104857600, 104857600, allow`

File: example/casbinconfig/permission_model.conf [hunk 6]
```diff
@@ -8,4 +8,4 @@ p = resourceName,resourceKind,operation,eft
 e=!some(where (p.eft == deny))
 
 [matchers]
-m= r.resourceName==p.resourceName && r.resourceKind==p.resourceKind&&r.operation==p.operation
\ No newline at end of file
+m= r.resourceName == p.resourceName && r.resourceKind == p.resourceKind && r.operation == p.operation
```
Comment (sagilio) [inline example/casbinconfig/permission_model.conf:11]: `m = r.resourceName == p.resourceName && r.resourceKind == p.resourceKind && r.operation == p.operation `

**Review by sagilio — CHANGES_REQUESTED** [review]
I need to prepare the environment for the review of other changes.

**Review by sagilio — APPROVED** [review]

**Pull request merged**